pom.xml

```diff
@@ -25,7 +25,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.dataformat</groupId>
             <artifactId>jackson-dataformat-csv</artifactId>
-            <version>2.9.4</version>
+            <version>2.9.6</version>
```

Comment:
This change would mean we move away from the `jackson-core` version provided by `connect-runtime`.  While a minor version change like this is probably OK, I would rather keep the versions in synch with the runtime to avoid any subtle incompatibilities.  Let's wait for the` connect-runtime` update.